component/airlock-xopenshift.jsonnet

```diff
@@ -34,9 +35,70 @@ local xopenshift_crds = [
   if std.member(std.objectFields(enabled_crds), basename(crd.metadata.name))
 ];
 
+local aggregate_crs(crdname) =
+  local rule = {
+    apiGroups: [ xopenshift_group ],
+    resources: [ basename(crdname) ],
+  };
+  [
+    kube.ClusterRole('%s-crdview' % crdname) {
```

Comment:
Never seen that one before, what is it used for?

Reply:
not sure, I copied the Airlock Microgatewya OLM-managed roles